canvas_modules/common-canvas/src/common-canvas/cc-right-flyout.jsx

```diff
@@ -19,10 +19,59 @@ import { connect } from "react-redux";
 import PropTypes from "prop-types";
 import Logger from "../logging/canvas-logger.js";
 
+const MIN_WIDTH = 300;
+const MAX_WIDTH_EXTEND_PERCENT = 0.7; // Should cover atmost 70% of available width
 class CommonCanvasRightFlyout extends React.Component {
 	constructor(props) {
 		super(props);
 		this.logger = new Logger("CC-RightFlyout");
+
+		this.onMouseDown = this.onMouseDown.bind(this);
+		this.onMouseMoveX = this.onMouseMoveX.bind(this);
+		this.onMouseUp = this.onMouseUp.bind(this);
+	}
+
+	onMouseDown(e) {
+		if (e.button === 0) {
+			document.addEventListener("mousemove", this.onMouseMoveX, true);
+			document.addEventListener("mouseup", this.onMouseUp, true);
+			this.posX = e.clientX;
+			this.startWidth = this.commonCanvasRightFlyout.offsetWidth;
+
+			e.preventDefault();
+		}
+	}
+
+	onMouseUp() {
+		document.removeEventListener("mousemove", this.onMouseMoveX, true);
+		document.removeEventListener("mouseup", this.onMouseUp, true);
+	}
+
+	onMouseMoveX(e) {
+		if (e.clientX) {
+			const newWidth = this.startWidth + (this.posX - e.clientX);
+			this.commonCanvasRightFlyout.style.width = `${this.limitWidth(newWidth)}px`;
+
+			// In case of Common Properties remove hardcoded width to allow content to take full width.
+			const righFlyoutChildNode = this.commonCanvasRightFlyout.childNodes[1];
+			if (righFlyoutChildNode && righFlyoutChildNode.className.includes("properties-right-flyout")) {
```

Comment:
This seems wrong since it's mixing common properties and common-canvas.

Reply:
Thanks, removed this code from CC.